conductor_api/src/conductor/base.rs

```diff
@@ -280,6 +280,7 @@ impl Conductor {
                 String::from("N3H_IPC_SOCKET") => String::from("tcp://127.0.0.1:*"),
                 String::from("N3H_LOG_LEVEL") => network_config.n3h_log_level.clone(),
             },
+            2000,
```

Comment:
Is 2000 long enough? Will people encounter timeout issues and not know how, or have a way, to set the timeout value?

Reply:
Thats a conductor level decision to make. You don't want to wait too long either, otherwise it will appear 'frozen'.